### PR TITLE
user navbar and route refactor

### LINKED ISSUE #93: Display invalid username/password error

### LINKED ISSUE #98: Move navigation into top bar & ditch dahsboard

### PR DESCRIPTION
**Pull request by nobane**
closes #98
closes #93 

This PR removes the notion of a "Dashboard" from the site. It also introduces dynamic routing. Page components have all been consolidated into a pages directory.

### CODE REVIEW

File: client/src/components/pages/provider/ProviderActiveAlertsPage.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        const description = (
+            'Here you can see alerts you have responded ' +
+            'to and any alerts within the past 2 days ' +
+            'that you have not responded to.'
+        );
+        return (
+            <AlertTable 
+              alerts={alerts}
+              role='provider'
+              title='Your Alerts'
```
Comment (justinnoah) [inline client/src/components/pages/provider/ProviderActiveAlertsPage.js:24]: Maybe this should just be "Alerts" or maybe "Available Alerts"? "Your Alerts" sounds like it's a list of alerts I've taken on the responsibility of.

Comment (nobane) [inline client/src/components/pages/provider/ProviderActiveAlertsPage.js:24]: Unresponded Alerts?

My browser says "unresponded" isn't a word :\

**Comment by justinnoah** [comment]
Capabilities are not listed on the "Edit Profile" page for providers. [Screencap](http://imgur.com/a/uu96V)

Edit:
Looks like this is the only issue  I'm coming across for the refactor.

File: client/src/components/pages/provider/ProviderActiveAlertsPage.js (hunk 1 above)
Comment (justinnoah) [inline client/src/components/pages/provider/ProviderActiveAlertsPage.js:24]: responded alerts are also listed aren't they? Either way, that's not holding this issue back. We may want to add a card on trello to have Jade do some deep bikeshedding on the project.

Comment (nobane) [inline client/src/components/pages/provider/ProviderActiveAlertsPage.js:24]: No it's just un-responded alerts. I'll go with just alerts. There will probably be another page that is like "Responded Alerts" or "Alert History" that shows their alert history.

**Comment by nobane** [comment]
@justinnoah Are you sure you have categories with services? If there are no services associated with the category, you won't see the category. [It is working fine for me](http://i.imgur.com/jnyENAt.png)

**Comment by justinnoah** [comment]
@spectralsun I had cats and services. but i may not have "reset" my environment entirely when reviewing.

**Pull request merged**